data-processing/tests/test_extract_definitions.py

```diff
@@ -0,0 +1,110 @@
+import pytest
+
+from entities.definitions.commands.detect_definitions import *
+
+
+@pytest.fixture(scope="module", autouse=True)
+def setup_model():
+    """
+    Only set up the model once, as it requires the time-consuming initialization
+    """
+    global model  # pylint: disable=global-statement
+    model = DefinitionDetectionModel()
+
+def test_term_definition_pairs(text, gold):
```

Comment:
nit pick---it feels like the method name is missing an action, like `test_extract_term_definition_pairs`